tests/ui/tsconfig.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
what is this file required for?

Reply:
the reference to document fails on ` const element = document.querySelector(selector);`, which can only be fixed by specifying a tsconfig localised to the test/ui folder